docs: LEAP-615: Mention spaces saving for Text tag

robot-ci-heartex opened the pull request:
Hi @hlomzik!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7675080188) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1682

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7675080188

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7675093678

caitlinwheeless committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7675197584

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7675210658

hlomzik commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7699150931)
> Successfully merged:  23 files changed, 443 insertions(+), 57 deletions(-)

hlomzik committed:
Merge branch 'develop' into 'fb-leap-615/spaces-in-text-results-docs'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7699150931

hlomzik committed:
Merge branch 'develop' into 'fb-leap-615/spaces-in-text-results-docs'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7913717929

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7913717929

hlomzik committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7913717929

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7913745674

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7914001534

hlomzik commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7915058793)
> Successfully merged: Already up to date.
